lib/kubeclient/kube_exception.rb

```diff
@@ -9,6 +9,10 @@ def initialize(error_code, message, response)
   end
 
   def to_s
```

Comment:
ideally I'd like to modify the `@message` in `initialize`, but that would be a bigger change ...